core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -53,6 +53,8 @@ object RequestChannel extends Logging {
     @volatile var apiLocalCompleteTimeMs = -1L
     @volatile var responseCompleteTimeMs = -1L
     @volatile var responseDequeueTimeMs = -1L
+    @volatile var apiRemoteCompleteTimeMs = -1L
```

Comment:
For follow-up (not necessary to include in this patch): would be great to have a test for  validating these times.

Also @junrao since you did this: can you clarify why these need to be volatile? Also how can `apiLocalTimeCompleteTimeMs` ever be negative? I read the comment but didn't quite follow.